### DISCUSSION TITLE
How do you use Komac?

### DISCUSSION THREAD

**Discussion by ero-qt** [microsoft/winget-pkgs#194870] [Q&A] [answered]
Hello.

I was attempting to update the package version for `AIMP.AIMP` to 5.30.2563 (#145278 for version 5.30.2541 was unactioned and closed) and I wanted to try out Komac for submitting the PR. However, the instructions and explanations are very lackluster.

1. What are the URLs I'm supposed to specify in `--urls`? Are they installer URLs? If so, do I provide the two URLs for 32-bit and 64-bit? Or do I provide the website URL? I do not understand.
2. What is the GitHub token it asks for when submitting the PR? Do I fork this repo and provide that? I do not understand.

cc @russellbanks

**Comment by russellbanks** [comment, marked as answer]
Hi,

This would be better suited to ask at [russellbanks/Komac](https://github.com/russellbanks/Komac) but as it's here:

1. The URLs are the installer URLs. So, yes, if there are two installers, a 32-bit one and a 64-bit one, you would provide an installer URL for each one.
2. You should fork `winget-pkgs` anyway so komac can make pull requests from that fork to the upstream `microsoft/winget-pkgs`. The GitHub token however is a [classic token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) with the `public_repo` scope checked.

The komac [ReadMe](https://github.com/russellbanks/Komac/blob/HEAD/README.md) has instructions for how to use komac, but I'll work on making them more robust and clear.

**Reply by ero-qt** [reply]
Unfortunately, https://github.com/russellbanks/Komac does not allow discussions. I didn't want to open an issue for a simple question.  

Komac has now warned me that there is already a closed PR for that exact version (should have checked) (awesome feature, by the way).  
It appears that `AIMP` is classified as riskware of some sort, which is understandable. I won't be opening a new PR in this case.

Thank you for the help.
